tools/src/wyvern/tools/typedAST/core/declarations/DelegateDeclaration.java

```diff
@@ -135,8 +136,9 @@ public DeclType genILType(GenContext ctx) {
 
 	@Override
 	public wyvern.target.corewyvernIL.decl.Declaration generateDecl(GenContext ctx, GenContext thisContext) {
-		// TODO Auto-generated method stub
-		return null;
+		String targetName = ((Variable)target).getName();
+		wyvern.target.corewyvernIL.decl.DelegateDeclaration ILDelegateDecl = new wyvern.target.corewyvernIL.decl.DelegateDeclaration(type.getILType(ctx), targetName);
```

Comment:
Can you name this starting with a lowercase variable?  I know it starts with IL which is an abbreviation, but name it differently somehow to conform to Java convention.